extensions/wikia/ArticleAsJson/ArticleAsJson.class.php

```diff
@@ -65,7 +65,8 @@ private static function renderImage( $media, $id ) {
 					'title' => $media['title'],
 					'fileUrl' => $media['fileUrl'],
 					'caption' => $media['caption'] ?? '',
-					'link' => $media['link'] ?? '',
+					'href' => $media['href'],
+					'isLinkedByUser' => $media['isLinkedByUser'],
```

Comment:
That would make sense but I don't agree it'd be easier to understand the logic behind it. "Default" is a pretty generic word and if you don't dive deep into the code you won't know what that means. `isLinkedByUser` is self-explanatory, which is more important than pretty :wink: